runtime/runtime/src/actions.rs

```diff
@@ -417,11 +430,21 @@ pub(crate) fn action_implicit_account_creation_transfer(
 }
 
 pub(crate) fn action_deploy_contract(
+    apply_state: &ApplyState,
     state_update: &mut TrieUpdate,
     account: &mut Account,
     account_id: &AccountId,
     deploy_contract: &DeployContractAction,
 ) -> Result<(), StorageError> {
+    #[cfg(feature = "protocol_feature_add_account_versions")]
+    recalculate_usage(
```

Comment:
protocol version also needs to be checked here

Reply:
Done